src/cmd/completion.go

```diff
@@ -0,0 +1,209 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/toolbox/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
```

Comment:
Note that this help string is likely not going to show up anywhere.  We explicitly try to consolidate our documentation in the manuals, to avoid fragmentation, and those don't come from here.  The strings in the Cobra `Command` instances are purely for developer documentation.

Secondly, there are no stability guarantees for the command line interface of this command.  It's purely an internal implementation detail that is liable to change anytime.  Any non-trivial documentation can give the wrong impression.

Third, the `completion` command comes with its own shell completion that's self-documenting and self-explanatory.  The documentation here is about how completion generally works in the shells, which isn't really Toolbx's domain.

Reply:
I agree with `Long` being redundant and should be removed.

While I like us relying mostly on manuals, I also think the use of items like `Use`, `Short` in the Cobra `Command` instances is also the right thing to do. We may not use them now but that might change if we decide to tackle [Project pickle](https://github.com/containers/toolbox/projects/4). Also manuals don't tend to be localized while code strings yes (and it's much easier to do).

Reply:
> I also think the use of items like Use, Short in the Cobra Command
> instances is also the right thing to do.

Like I said, they are fine for developer documentation and a few other specific things.  However, not when it comes to any non-trivial amount of text.  Those are harder to keep synchronized across multiple places.

> We may not use them now but that might change if we decide to
> tackle [Project pickle](https://github.com/containers/toolbox/projects/4).

We will cross that bridge when we come to it.  :)

> Also manuals don't tend to be localized while code strings yes

```
$ ls /usr/share/man
ca  es  id  ...  pt  ...
cs  fr  it  ...  pt_BR  ...
```

> Also, the text in Short is used in Cobra-generated completions scripts.

Yes, I agree.  That's the string in `Short` matches the title of the corresponding manual.  :)

Reply:
Resolved, since `Long` was dropped in https://github.com/containers/toolbox/pull/1055 (and https://github.com/containers/toolbox/pull/1113).